.github/workflows/sharness.yml

```diff
@@ -0,0 +1,118 @@
+name: '[EXPERIMENTAL] Sharness'
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  runner:
+    if: github.repository == 'ipfs/kubo' || github.event_name == 'workflow_dispatch'
+    uses: ipfs/kubo/.github/workflows/runner.yml@ci/move-to-github-actions # TODO: change to master
```

Comment:
TODO for @galargh ahead of the merge. Not changing it yet because there is no `ipfs/kubo/.github/workflows/runner.yml` in `master` yet so the CI in this PR would break.

Reply:
Actually, I'll change it in a subsequent PR. That way this one will be green when we merge it and `master` will be green after the merge.